metricbeat/docs/configuring-howto.asciidoc

```diff
@@ -18,14 +18,18 @@ For a complete description of all {beatname_uc} configuration options, see
 
 A Metricbeat configuration contains a list of modules. Each module contains the following config options:
 
-* module: Name of the module to run
-* metricsets: List of metricsets of the module which should be executed
-* hosts: List of hosts that the metricsets should connected to. For some modules, such as the system module, this is not required. For the details how to define the hosts string for each module, please check the documentation of the module.
-* period: How often in seconds the metricsets are executed. If a system is not reachable it will return an error for each period.
-* enabled: Modules are enabled (set to `enabled: true`) by default. To disable a module, specify `enabled: false`.
-* fields: Fields is optional. Here you can define additional fields which should be sent with the metricset event
-* tags: Tags is optional. Here you can define additional tags which should be sent with the metricset event
-* filters: With filters you can reduce the amount of data is sent. For details on the available filter options check (link to filters).
+* `module`: The name of the module to run. For documentation about each module, see the <<metricbeat-modules>> section.
+* `metricsets`: A list of metricsets to execute. For a list of available metricsets, see the documentation for the module.
+* `enabled`: Specifies whether the module is enabled. 
+* `period`: How often the metricsets are executed. 
+* `hosts`: A list of hosts to fetch information from. For some modules, such as the system module, this setting is not required.
+* `fields`: A dictionary of fields that will be sent with the metricset event. This setting is optional. 
+* `filters`: With filters you can reduce the amount of data that is sent in the event. For more about the available filter options, see <<filtering-data>>. This setting is optional. 
+* `tags`: A list of tags that will be sent with the metricset event. This setting is optional. 
```

Comment:
@ruflin They should be listed in the order that they appear in the metricbeat.full.yml file.